tests/streamlit_ui_tests/utils.ts

```diff
@@ -48,6 +48,15 @@ export const checkColumnCellsHaveExpectedValues = async (mito: FrameLocator, col
     }
 }
 
+export const fillInput = async (mito: FrameLocator, title: string, value: string): Promise<void> => {
+    await mito.locator('.spacing-row', { hasText: title }).locator('input').fill(value);
+}
+
+export const updateSelectedValue = async (mito: FrameLocator, title: string, value: string): Promise<void> => {
+    await mito.locator('.spacing-row', { hasText: title }).locator('.select-text').click();
+    await mito.locator('.mito-dropdown-item', { hasText: value }).click();
+}
```

Comment:
```suggestion
export const fillInput = async (mito: FrameLocator, inputLabel: string, value: string): Promise<void> => {
    await mito.locator('.spacing-row', { hasText: inputLabel }).locator('input').fill(value);
}

export const updateSelectedValue = async (mito: FrameLocator, selectLabel: string, value: string): Promise<void> => {
    await mito.locator('.spacing-row', { hasText: selectLabel }).locator('.select-text').click();
    await mito.locator('.mito-dropdown-item', { hasText: value }).click();
}
```

Reply:
Maybe this is clearer to differentiate between title tags of elements (which these inputs and selects often have) vs the label in the row